modules/opencensus/pom.xml

```diff
@@ -116,4 +116,27 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>copy-libs</id>
+                        <phase>test-compile</phase>
+                        <goals>
+                            <goal>copy-dependencies</goal>
+                        </goals>
+                        <configuration>
+                            <excludeGroupIds>org.gridgain</excludeGroupIds>
```

Comment:
Do we really need this? It seems all _vendor-specific_ things must be excluded from the pom.

Reply:
My fault. Fixed. `<excludeGroupIds>org.gridgain</excludeGroupIds>` -> `<excludeGroupIds>org.apache.ignite</excludeGroupIds>`